src/H5public.h

```diff
@@ -386,6 +386,9 @@ typedef struct H5_alloc_stats_t {
     size_t             peak_alloc_blocks_count;  /**< Peak # of blocks allocated */
 } H5_alloc_stats_t;
 
+/* Library shutdown callback */
+typedef void (*H5_atclose_func_t)(void *ctx);
```

Comment:
Do typedefs get doxygen comments?

Reply:
Seems like it (based on H5Fpublic.h).   Added.